### PR TITLE
Remove unused parameters.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Extracted from #41510 to make that PR easier to review.

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
Tagging @mavasani

**Comment by CyrusNajmabadi** [comment]
@mavasani , this PR in particular doesn't even seem like it needs to be part of the "move helpers down" change.  It's purely a cleanup change.  Certainly good o have.  But it's just extra stuff to review as part of the large change that doesn't seem necessary.

File: src/Features/VisualBasic/Portable/SimplifyThisOrMe/VisualBasicSimplifyThisOrMeCodeFixProvider.vb [hunk 1]
```diff
@@ -23,24 +23,22 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.SimplifyThisOrMe
         End Function
 
         Protected Overrides Function Rewrite(semanticModel As SemanticModel, root As SyntaxNode, memberAccessNodes As ISet(Of MemberAccessExpressionSyntax)) As SyntaxNode
```
Comment (jasonmalinowski) [inline src/Features/VisualBasic/Portable/SimplifyThisOrMe/VisualBasicSimplifyThisOrMeCodeFixProvider.vb]: @CyrusNajmabadi Can we get rid of semanticModel being passed down at any higher level?

Comment (CyrusNajmabadi) [reply src/Features/VisualBasic/Portable/SimplifyThisOrMe/VisualBasicSimplifyThisOrMeCodeFixProvider.vb]: Will look into doing that!

**Comment by CyrusNajmabadi** [comment]
Thanks!

**Pull request merged**